ius-ecclesiae.csl

```diff
@@ -0,0 +1,683 @@
+        <multiple>herausgegeben von</multiple>
+      </term>
+      <term name="editor" form="short">
+        <single>Hg.</single>
+        <multiple>Hgg.</multiple>
+      </term>
+      <term name="collection-editor" form="verb">
+        <single>herausgegeben von</single>
+        <multiple>herausgegeben von</multiple>
+      </term>
+      <term name="collection-editor" form="short">
+        <single>Hg.</single>
+        <multiple>Hgg.</multiple>
+      </term>
+      <term name="translator" form="short">
+        <single>Übers.</single>
+        <multiple>Übers.</multiple>
+      </term>
+      <term name="volume" form="short">
+        <single>Bd.</single>
+        <multiple>Bd.</multiple>
+      </term>
+      <term name="page" form="short">
+        <single>S.</single>
+        <multiple>S.</multiple>
+      </term>
+      <term name="paragraph" form="short">
+        <single>§</single>
+        <multiple>§§</multiple>
+      </term>
+      <term name="in">in</term>
+      <term name="cited" form="short">cit.</term>
+      <term name="accessed" form="long">zuletz besucht</term>
+      <term name="presented at">in</term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <names variable="author">
+      <name initialize-with=". " delimiter=" &#8211; " delimiter-precedes-last="always" font-variant="small-caps"/>
+    </names>
+  </macro>
+  <macro name="author-sort">
+    <names variable="author">
+      <name name-as-sort-order="all" initialize-with=". " delimiter=" &#8211; " delimiter-precedes-last="always" font-variant="small-caps"/>
+    </names>
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <name initialize-with=". " delimiter=" &#8211; " delimiter-precedes-last="always" font-variant="small-caps"/>
+      <label form="short" prefix=" (" suffix=")"/>
+    </names>
+  </macro>
+  <macro name="editor-sort">
+    <names variable="editor">
+      <name name-as-sort-order="all" initialize-with=". " delimiter=" &#8211; " delimiter-precedes-last="always" font-variant="small-caps"/>
+      <label form="short" prefix=" (" suffix=")"/>
+    </names>
+  </macro>
+  <macro name="translator">
+    <names variable="translator">
+      <name initialize-with=". " delimiter=" &#8212; " delimiter-precedes-last="always" font-variant="small-caps"/>
+      <label form="short" prefix=" (" suffix=")"/>
+    </names>
+  </macro>
+  <macro name="translator-sort">
+    <names variable="translator">
+      <name name-as-sort-order="all" initialize-with=". " delimiter=" &#8212; " delimiter-precedes-last="always" font-variant="small-caps"/>
+      <label form="short" prefix=" (" suffix=")"/>
+    </names>
+  </macro>
+  <macro name="creator">
+    <choose>
+      <if variable="author editor" match="all">
+        <text macro="author"/>
+      </if>
+      <else-if variable="author">
+        <text macro="author"/>
+      </else-if>
+      <else>
+        <text macro="editor"/>
+      </else>
+    </choose>
+  </macro>
```

Comment:
And just so you're aware, you could also use:

```xml
  <macro name="creator">
    <names variable="author">
      <name initialize-with=". " delimiter=" &#8211; " delimiter-precedes-last="always" font-variant="small-caps"/>
      <label form="short" prefix=" (" suffix=")"/>
      <substitute>
        <names variable="editor"/>
      </substitute>
    </names>
  </macro>
```

See also http://docs.citationstyles.org/en/stable/specification.html#substitute. I'm not sure if it would help simplify your style, but we rely on `<substitute/>` in a lot of CSL styles. This macro will print the "author" names, or, if there aren't any, the "editor" names. There is no term defined in the locale files for "author", so the macro above will only print a label for "editor".